.github/workflows/swift.yml

```diff
@@ -19,24 +19,19 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
-
-      - name: Install SwiftLints
-        run: brew install swiftformat
-
-      - name: SwiftLint
+      - name: Install swift-format
+        run: brew install swift-format
+      - name: swift-format Lint
```

Comment:
I would consider running lint and format in the same job, as separate steps. The number of macOS concurrent instances is limited and we are already using too many.